packages/hayabusa.vm/tools/chocolateyinstall.ps1

```diff
@@ -4,10 +4,9 @@ Import-Module vm.common -Force -DisableNameChecking
 $toolName = 'hayabusa'
 $category = 'Forensic'
 
-$version = '2.11.0'
```

Comment:
I did it this way so that it could be updated automatically by the pipeline (but I can be wrong)

Reply:
It won't because the executable name also includes the version. But we could modify the `update_package.py script to replace the version in the whole `chocolateyinstall.ps1` text instead of only in the `url` and this installer to use the whole executable name:
`$executableName = `hayabusa-2.15.0-win-x64.exe"`

Then the CI should update the package.

I am ok with this change, even if we do not implement this change in this PR.

Reply:
It's the same as leaving as modified and editing  `update_package.py` to replace all occurrence of the version ? I can do it in this PR :)

Reply:
I would suggest leaving it as it is in this PR and create a new PR to modify `update_package.py` to replace all occurrence of the version so that we can review the two changes independently. :wink: